app/helpers/reactjs_helper.rb

```diff
@@ -45,17 +49,31 @@ def js_tags_for(requested_plugins)
     end
   end
 
-  def js_tags_for_global_files(requested_plugins)
+  def global_js_tags(requested_plugins)
     requested_plugins.map do |plugin|
       plugin[:files].map do |file|
         javascript_include_tag(*webpack_asset_paths(plugin[:id].to_s + ":#{file}", :extension => 'js'), :defer => "defer")
       end
     end
   end
 
+  def global_css_tags(requested_plugins)
+    requested_plugins.map do |plugin|
+      plugin[:files].map do |file|
+        stylesheet_link_tag(*webpack_asset_paths(plugin[:id].to_s + ":#{file}", :extension => 'css'))
+      end
```

Comment:
This looks  very weird to me. `files` is a list of global **JS** files, not CSS if I read the API right. Why does this work?

Reply:
Because webpack allows requireing css in js files, if you do so, it generates a css file with the same name as the currently generated js file.
What I do wander is: what about js files that do not have css requires, so they don't generate this accompaniing css file, won't this be a miss (404)?